data/pidgin/livePage/c7p765ynk9qt.json

```diff
@@ -0,0 +1,104 @@
+{
+  "data": {
+    "title": "Pidgin test 2",
+    "description": "Pidgin test 2 - the description",
+    "language": "pcm",
+    "headerImage": null,
+    "promoImage": null,
+    "home": "pidgin",
+    "liveTextStream": {
+      "id": "8E1A80B519D1451FBF5DF6AB029B8B1C"
+    },
+    "sportDataEvent": {
+      "id": null
+    },
+    "article": {
+      "id": null
+    },
+    "mediaCollections": [
+      {
+        "id": null
+      }
+    ],
+    "seo": {
+      "seoTitle": "Pidgin test 2",
+      "seoDescription": "Pidgin test 2 - the description"
+    },
+    "endDateTime": "2023-04-06T10:21:00.000Z",
+    "startDateTime": "2023-04-05T10:22:00.000Z",
+    "analytics": {
+      "page": {
+        "name": "live_coverage.c7p765ynk9qt.page",
+        "contentId": "urn:bbc:tipo:topic:c7p765ynk9qt"
+      }
+    },
+    "summaryPoints": {
+      "id": "urn:bbc:optimo:asset:c5x3899j84ro",
```

Comment:
Heya, I've a quick question. I can see here you've 'inserted' the response into this `summaryPoints` object which is inside of the `data` object.

Is there a downside in nesting the 'summaryPoints' response inside the `liveExperience` response? Or is this always the most logical thing to do?

It's a question I raised while working on adding the `liveTextStream`. I wasn't sure if nesting responses could cause issues if one API fails. (For more context:  I can see with articles we have appended topStories, features, mostRead etc outside of the 'article' response.)

Reply:
Ello, I think it just comes down to to our preference when it comes to designing our API, I don't think there's a downside per say. The only reason I did it this way, is because the `summaryPoints` key [already existed](https://github.com/bbc/fabl-modules/blob/e3f3c369779ef2f29b0e1f47f3197441767809ed/modules/application/simorgh-bff/src/fixtures/pidgin/liveExperienceFixture.json#L24-L25), so I just [tagged the content on](https://github.com/bbc/fabl-modules/blob/e3f3c369779ef2f29b0e1f47f3197441767809ed/modules/application/simorgh-bff/src/pages/live/index.ts#L91-L96) when constructing the response. 

For the articles, it was an [API design choice](https://jira.dev.bbc.co.uk/browse/WSTEAM1-59#:~:text=how%20this%20works.-,the%20data%20view%20model,-Planning%20out%20the) because the main article data is separate in terms of layout to the secondary data which is mostly found in the secondary column.